packages/libs/src/services/solana/SolanaWeb3Manager.ts

```diff
@@ -689,11 +708,10 @@ export class SolanaWeb3Manager {
     )
 
     const data = `${type}:${id}:${blocknumber}:${purchaserUserId}`
-
     const memoInstruction = new TransactionInstruction({
       keys: [
         {
-          pubkey: senderAccount,
+          pubkey: this.feePayerKey,
```

Comment:
We can't sign the memo with the feepayer for coinflow, since we aren't relaying those. Maybe we need to parameterize this...